nearcore/Cargo.toml

```diff
@@ -111,6 +111,11 @@ protocol_feature_fix_contract_loading_cost = [
   "near-vm-runner/protocol_feature_fix_contract_loading_cost",
 ]
 protocol_feature_flat_state = ["near-store/protocol_feature_flat_state", "near-chain/protocol_feature_flat_state", "node-runtime/protocol_feature_flat_state"]
+protocol_feature_delegate_action = [
```

Comment:
I think this should be added to nightly here in nearcore/Cargo.toml, otherwise our CI testing with `features=nightly` will not enable your new feature. (for flat state just a line above, this is intentional, all other protocol features should be enabled in nightly)

Reply:
Done